package.json

```diff
@@ -81,10 +81,11 @@
     "@mojaloop/central-services-health": "10.1.0",
     "@mojaloop/central-services-logger": "9.5.1",
     "@mojaloop/central-services-metrics": "9.5.0",
-    "@mojaloop/central-services-shared": "9.5.5",
+    "@mojaloop/central-services-shared": "10.1.0",
     "@mojaloop/central-services-stream": "9.5.0",
     "@mojaloop/event-sdk": "9.5.2",
     "@mojaloop/forensic-logging-client": "8.3.0",
+    "@mojaloop/sdk-standard-components": "10.1.0",
```

Comment:
I don't think we should be importing the `sdk-standard-components` in both the central-services-shared and in this library that ALSO imports both libraries.

Can we not inject the JWSSigner?

Reply:
Yes, we inject JWSSigner into `central-services-shared` 
In `central-services-shared` it's imported only for unit testing as dev dependency